### PR TITLE
feat: show when no browser wallet found

### LINKED ISSUE #738: Add "No wallet detected in this browser" placeholder in wallet select menu
Folks have multiple browsers and it happens to the best that they open the wrong browser and wonder "why metamask doesn't show in the menu"

EDIT: also applies when I use a Web3 wallet that is not supported by Boltz Web App yet, see https://github.com/BoltzExchange/boltz-web-app/issues/743

### PR DESCRIPTION
**Pull request by michael1011**
Closes #738

@kilrau needs translations

image

### CODE REVIEW

**Review by jackstar12 — APPROVED** [review]
Looks better without the warning imo (see comments):

image

File: src/components/ConnectWallet.tsx [hunk 1]
```diff
@@ -90,6 +98,15 @@ const Modal = (props: {
                     <IoClose />
                 </span>
                 <hr class="spacer" />
+                <Show when={!hasBrowserWallet()}>
+                    <hr />
+
+                    <div class="no-browser-wallet">
+                        <Warning />
+                        <h3>{t("no_browser_wallet")}</h3>
+                    </div>
```
Comment (jackstar12) [inline src/components/ConnectWallet.tsx:107]:
```suggestion
                    <h3 class="no-browser-wallet">{t("no_browser_wallet")}</h3>
```

File: src/components/Warning.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const Warning = () => {
+    return <p style={{ "font-size": "46px", margin: "0" }}>⚠️</p>;
+};
+
+export default Warning;
```
Comment (jackstar12) [inline src/components/Warning.tsx:5]:
```suggestion

```

**Comment by kilrau** [comment]
> Looks better without the icon imo (see comments):
> 
> image

Agree

**Review by kilrau — APPROVED** [review]
tACK

**Pull request merged**